packages/openapi/src/middleware.test.ts

```diff
@@ -172,6 +172,41 @@ describe('OpenAPI Validator', (): void => {
       expect(next).toHaveBeenCalled()
     })
 
+    test('returns 500 with additional properties', async (): Promise<void> => {
+      const ctx = createContext(
+        {
+          headers: {
+            Accept: 'application/json',
+            'Content-Type': 'application/json'
+          }
+        },
+        {}
+      )
+      addTestSignatureHeaders(ctx)
+      const next = jest.fn().mockImplementation(() => {
+        ctx.response.body = {
+          pagination: { hasNextPage: false, hasPreviousPage: false },
+          result: [
+            {
+              id: uuid(),
+              walletAddress: `https:ilp.rafiki.money/${uuid()}`,
+              completed: false,
+              receivedAmount: { value: '0', assetCode: 'USD', assetScale: 2 },
+              createdAt: '2022-03-12T23:20:50.52Z',
+              updatedAt: '2022-04-01T10:24:36.11Z',
+              additionalProp: 'disallowed'
+            }
+          ]
+        }
+      })
+      await expect(validateListMiddleware(ctx, next)).rejects.toMatchObject({
+        status: 500,
+        message:
+          'response.result.0 must NOT have additional properties: additionalProp'
```

Comment:
I'm guessing this is a test for an older spec?

Reply:
When I pushed this test up it was passing locally but failed here until I update my branch (the gh action used the new one from main somehow?). I added this because I moved `additionalProperties: false` to the `GET /incoming-payments` response (the list) from the `incoming-payment` types itself. So it's testing that, which should preserve existing functionality.